src/tutorials/lua_basics_lightbulb.md

```diff
@@ -130,50 +211,61 @@ Your rotations may be different, but will most likely look like this:
 
 |  On / Off | X | Y | Z |
 | --- | --- | --- | --- |
-|  On | -0 | -60 | 0 |
-|  Off | -180 | -60 | 180 |
+|  On | 0 | -60 | 0 |
+|  Off | 0 | -130 | 0 |
 
 ## Scripting the Rotation
 
 ### Create Variables for Each Rotation
 
-Now that you know the rotations of the switch for "on" and "off", you can save them as variables. On the next line of your `LightToggleScript` add these lines:
+Now that you know the rotations of the switch for "on" and "off", you can save them as variables. On the next line of your `LightSwitchToggle` add these lines:
 
 ```lua
-local onRotation = Rotation.New(0, -60, -0)
-local offRotation = Rotation.New(-180, -60, 180)
+local ROTATION_ON = Rotation.New(0, -60, 0)
+local ROTATION_OFF = Rotation.New(0, -130, 0)
 ```
 
 - `Rotation.New` means we are telling the script to rotate our object to a new set of coordinates.
-- `(0, 60, 0)` are the x, y, and z coordinates (respectively) of where we want our switch to rotate to. We want to rotate our switch up along the y-axis by 90 degrees.
+- `(x, y, z)` are the coordinates (respectively) of where we want our switch to rotate to. We want to rotate our switch up along the y-axis by 90 degrees.
 
 !!! note
     You will almost always use `Rotation.New` when rotating an object, but there is also a predefined Rotation called `Rotation.ZERO` which will rotate the object to `0, 0, 0`.
     [Learn more about Rotations on the Core API](../api/rotation.md) page.
 
-### Make the Switch Move with a Script
+!!! note
+    It is convention that constants, variables that will not be altered, are in **SCREAMING_SNAKE_CASE**, meaning all caps with **_** between each word.
```

Comment:
Take out "It is convention", and say that you/we used it for the Rotation variables as a way to signal that it never changes.

Also, sorry to kill joy, because that name is hilarious, but CAPS_CASE. 

and in the explanation change "caps" to "upper-case letters" or "capital letters" (apparently they're both used pretty equally) in case they don't get it from context.